lib/api/BackbeatAPI.js

```diff
@@ -113,6 +135,114 @@ class BackbeatAPI {
         });
     }
 
+    /**
+     * Get replication backlog in bytes and count
+     * @param {function} cb -callback(error, data)
+     * @return {undefined}
+     */
+    backlog(cb) {
+        async.series([
+            next => this._queryStats(OPS, next),
+            next => this._queryStats(BYTES, next),
+        ], (err, res) => {
+            if (err) {
+                this._logger.trace('error occurred getting backlog');
+                return cb(errors.InternalError);
+            }
+            const data = {
+                backlog: {
+                    details: 'Replication backlog in bytes and count',
```

Comment:
See comment below for rephrasing more explicitly.

Personally I'm not fond of "details", I would rather use "description" which is more descriptive (sorry :sheep: )

As a side note, in the ring we ran into issues because it has so many configuration values and statistics and each one of them carries a lengthy description, leading to many superflous bytes transmitted for every statistics polling that it became an issue on its own. But for now it should not be a problem for us, and it's still good to have the description close to the data transmitted, if it eventually becomes an issue we can rethink this at a later time.